src/app/tree-grid/tree-grid-column-selection-toolbar/tree-grid-column-selection-toolbar.component.scss

```diff
@@ -0,0 +1,53 @@
+// custom-grid-paging-style.component.scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+
+$white: #fff;
+$blue: #4567bb;
+
+$color-palette: igx-palette($primary: $white, $secondary: $blue);
+
+$grid-theme: igx-grid-theme(
+    $cell-editing-background: $blue,
+    $cell-active-border-color: $white,
+    $cell-edited-value-color: $white,
+    $edit-mode-color: igx-color($color-palette, "secondary", 200)
+);
+
+$checkbox-theme: igx-checkbox-theme(
+    $empty-color: igx-color($color-palette, "secondary", 200),
+    $fill-color: $white,
+    $tick-color: $blue
+);
+
+$datepicker-theme: igx-calendar-theme(
+    $date-selected-text-color: $white,
+    $date-selected-background: $blue
+);
+
+$input-theme: igx-input-group-theme(
+    $palette: igx-palette,
+    $filled-text-color: $white,
+    $focused-text-color: $white,
+    $idle-text-color: $white,
+    $idle-bottom-line-color: $white,
+    $focused-bottom-line-color: $white,
+    $interim-bottom-line-color: $white,
+    $hover-bottom-line-color: $white,
+    $box-background: $blue
+);
+
+:host {
+    padding: 3px 10px;
+    ::ng-deep {
+        .igx-grid__tbody{
+            @include igx-input-group($input-theme);
+        }
+        @include igx-checkbox($checkbox-theme);
+        @include igx-calendar($datepicker-theme);
+        @include igx-grid($grid-theme);
+    }
+}
```

Comment:
No point in this. The grid styling should be covered by another more generic topic that covers the grid.